services/ui-src/src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,259 @@
+import { Link as RouterLink } from "react-router-dom";
+// components
```

Comment:
Brendan mentioned not doing this pattern of comments for imports 😬 I went ahead and updated my branch to reflect that update too!